src/deskolem.ml

```diff
@@ -229,9 +267,12 @@ let rec get_x : term -> term Bindlib.var list * term = fun t ->
     match s with
     |Symb({sym_name = "∀"; _})     ->
```

Comment:
This is not correct to test symbol name equality. You should use Term.is_symb cfg.symb_... instead.